chronos/transports/ipnet.nim

```diff
@@ -36,73 +36,85 @@ type
 proc toNetworkOrder(mask: IpMask): IpMask {.inline.} =
```

Comment:
ideally, we'd remove this function and only ever create `IpMask` instances in one order (network?) - this way, we have a way of knowing which order the data is in by type, without exceptions